platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/spi/info/NodeInfo.java

```diff
@@ -84,4 +85,29 @@ default boolean zeroStake() {
      * @return the stake weight
      */
     long stake();
+
+    /**
+     * The signing x509 certificate bytes of the member
+     * @return the signing x509 certificate bytes
+     */
+    default Bytes sigCertBytes() {
+        return Bytes.EMPTY;
+    }
+
+    /**
+     * The host name of this node, as known by the internal world. This is an IP address.
+     *
+     * @return The host name (IP Address) of this node
+     */
+    default String internalHostName() {
+        return null;
+    }
```

Comment:
This is not optional right? SO, why should we have default method on the interface

Reply:
Since NodeInfo is an interface and there are other implementation classes which do not use these methods. Instead of trying to add to those classes, I used default here.

Reply:
All the implementations are test classes. It might be confusing to add a default for non-optional values

Reply:
SelfNodeInfoImpl is not test class though.

Reply:
Changed not to use default methods and implemented all needed classes methods.